packages/airnode-node/src/evm/fulfillments/api-calls.ts

```diff
@@ -316,15 +316,20 @@ export async function estimateGasAndSubmitFulfill(
   }
 
   const [airnodeRrpGasCost, fulfillmentCallGasCost] = estimateGasData;
-  const totalCost = airnodeRrpGasCost.add(fulfillmentCallGasCost);
+  const subTotalCost = airnodeRrpGasCost.add(fulfillmentCallGasCost);
+  // https://github.com/ethereum/EIPs/issues/114
+  // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-150.md
+  const eip150GasReserve = subTotalCost.div(64);
```

Comment:
Shouldn't we divide by 63 since our estimate is 63/64 of the true amount required?

Reply:
Right, good catch 👏🏻

Reply:
https://github.com/api3dao/airnode/pull/2001/commits/36d4325c7b45f5bf9a149b7bc68a626ee8ecf573